enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DefaultServiceEnricher.java

```diff
@@ -141,14 +135,34 @@ private void addServices(KubernetesListBuilder builder) {
         }
     }
 
+    private String getAppName(){
+        try {
+            if (getContext().getProjectClassLoaders().isClassInCompileClasspath(true)) {
+                Properties properties = SpringBootUtil.getSpringBootApplicationProperties(getContext().getProjectClassLoaders().getCompileClassLoader());
+                return properties.getProperty("spring.application.name");
```

Comment:
@ro14nd @lordofthejars @rohanKanojia can you provide other property names that we should consider for the name ? 

* spring.app.name
* app.name
* spring.service.name ?

WDYT ?

Reply:
we should stick to standard spring properties, which is `spring.application.name` in this case. If there are other ways there, then we can add them here as well.

Reply:
yeah, that's what I meant.. 

I think `spring.app.name` should be considered as well

Reply:
@theexplorist can you add a `getProperty("spring.app.name")` as well? Even in #872 its mentioned `spring.app.name` .